### PR TITLE
Refactor UI architecture to avoid repetition

### PR DESCRIPTION
**Pull request by ghengeveld**
Introduces a Screen component that wraps every screen so that we don't need to repeat the footer in every place, among other things. I also fixed some UI bugs that came with Storybook 8's updated component library.

<details>
  <summary>📦 Published PR as canary version: <code>1.1.5--canary.184.5d57486.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromatic-com/storybook@1.1.5--canary.184.5d57486.0
  # or 
  yarn add @chromatic-com/storybook@1.1.5--canary.184.5d57486.0
  ```
</details>

### CODE REVIEW

File: src/screens/LinkProject/LinkingProjectFailed.stories.ts [hunk 1]
```diff
@@ -10,6 +13,9 @@ const meta = {
     projectId: "Project:abc123",
     configFile: "chromatic.config.json",
   },
+  decorators: [
+    storyWrapper(AuthProvider, () => ({ value: { accessToken: "token", setAccessToken: fn() } })),
+  ],
```
Comment (thafryer) [inline src/screens/LinkProject/LinkingProjectFailed.stories.ts:18]: Is this used everywhere? You would get more reuse by making it a global decorator in `.storybook/preview.tsx`

Comment (ghengeveld) [reply src/screens/LinkProject/LinkingProjectFailed.stories.ts:18]: Good call.

File: src/screens/VisualTests/VisualTests.stories.tsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ import {
 import { delay, HttpResponse } from "msw";
 import React from "react";
 
+import { AuthProvider } from "../../AuthContext";
```
Comment (thafryer) [inline src/screens/VisualTests/VisualTests.stories.tsx:19]: Is this import necessary? It seems unused.

**Review by thafryer — APPROVED** [review]
LGTM. I left one small note. it's interesting that linter did not pick it up.

**Review by andrewortwein — APPROVED** [review]
🧑🏻‍🏭🛠️👍🏻

**Pull request merged**